CanonicalLayouts/README.md

```diff
@@ -10,16 +10,16 @@ Clone the [user-interface-samples](https://github.com/android/user-interface-sam
 
 ## Available Canonical layouts and related implementations
 - List-Detail
-    - [Activities](/list-detail-activity-embedding)
-    - [Views and Fragments](/list-detail-sliding-pane)
-    - [Jetpack Compose](/list-detail-compose)
+    - [Activities](https://github.com/android/user-interface-samples/tree/main/CanonicalLayouts/list-detail-activity-embedding)
+    - [Views and Fragments](https://github.com/android/user-interface-samples/tree/main/CanonicalLayouts/list-detail-sliding-pane)
+    - [Jetpack Compose](https://github.com/android/user-interface-samples/tree/main/CanonicalLayouts/list-detail-compose)
```

Comment:
>Links starting with / will be relative to the repository root. You can use all relative link operands, such as ./ and ../.

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes#relative-links-and-image-paths-in-readme-files

Maybe `./` is sufficient?

Reply:
It looks like it works with the `./` you suggested. Updated the PR